esmvalcore/cmor/_fixes/fix.py

```diff
@@ -197,21 +198,25 @@ def get_fixes(project, dataset, mip, short_name, extra_facets=None):
         return fixes
 
     @staticmethod
-    def get_fixed_filepath(output_dir, filepath):
+    def get_fixed_filepath(
+        output_dir: str | Path,
+        filepath: str | Path,
+    ) -> str:
```

Comment:
Would it be possible to make the return value a `Path`? I know this is not strictly backward compatible, but it would be nice to try and switch over to using `Path`s instead of `str` when we mean a path everywhere in the tool.

Reply:
I also thought about this, but decided against it because of backwards-compatibility. I will change it, but without a deprecation warning.